src/main.py

```diff
@@ -96,6 +96,8 @@
     c = json.load(json_file3)
 with open('standards/policy_rule_d.json') as json_file4:
     d = json.load(json_file4)
+with open('standards/policy_rule_issuer.json') as json_file5:
```

Comment:
We should avoid loading policies by default. I suggest to remove this and make it part of tests instead. 
The PDP should start in an empty state.